k/kong/kong-3.3.0.patch

```diff
@@ -0,0 +1,412 @@
+diff --git a/Makefile b/Makefile
+index 917e0070b..7f231a5d3 100644
+--- a/Makefile
++++ b/Makefile
+@@ -39,7 +39,7 @@ endif
+ 
+ ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+ KONG_SOURCE_LOCATION ?= $(ROOT_DIR)
+-GRPCURL_VERSION ?= 1.8.5
++GRPCURL_VERSION ?= 1.8.7
+ BAZLISK_VERSION ?= 1.16.0
+ BAZEL := $(shell command -v bazel 2> /dev/null)
+ VENV = /dev/null # backward compatibility when no venv is built
+@@ -67,9 +67,7 @@ build-kong: check-bazel
+ build-venv: check-bazel
+ 	$(eval VENV := bazel-bin/build/$(BUILD_NAME)-venv.sh)
+ 
+-	@if [ ! -e bazel-bin/build/$(BUILD_NAME)-venv.sh ]; then \
+-		$(BAZEL) build //build:venv --verbose_failures --action_env=BUILD_NAME=$(BUILD_NAME); \
+-	fi
++	$(BAZEL) build //build:venv --verbose_failures --action_env=BUILD_NAME=$(BUILD_NAME)
+ 
+ install-dev-rocks: build-venv
+ 	@. $(VENV) ;\
+@@ -86,7 +84,6 @@ install-dev-rocks: build-venv
+ dev: build-venv install-dev-rocks bin/grpcurl
+ 
+ build-release: check-bazel
+-	$(BAZEL) build clean --expunge
+ 	$(BAZEL) build //build:kong --verbose_failures --config release
+ 
+ package/deb: check-bazel build-release
+diff --git a/build/nfpm/repositories.bzl b/build/nfpm/repositories.bzl
+index cc719072e..8939bc929 100644
+--- a/build/nfpm/repositories.bzl
++++ b/build/nfpm/repositories.bzl
+@@ -13,8 +13,8 @@ def _nfpm_release_select_impl(ctx):
+ 
+     if os_arch == "aarch64":
+         os_arch = "arm64"
+-    elif os_arch == "amd64":
+-        os_arch = "x86_64"
++    elif os_arch == "ppc64le":
++        os_arch = "ppc64le"
+     else:
+         fail("Unsupported arch %s" % os_arch)
+ 
+@@ -36,15 +36,16 @@ nfpm_release_select = repository_rule(
+ 
+ def nfpm_repositories():
+     npfm_matrix = [
+-        ["linux", "x86_64", "4c63031ddbef198e21c8561c438dde4c93c3457ffdc868d7d28fa670e0cc14e5"],
```

Comment:
why do we need to remove the x86_64 references here and in subsequent lines? can't we just introduce Power support?